src/applications/edu-benefits/components/GetFormHelp.jsx

```diff
@@ -5,27 +5,16 @@ function GetFormHelp() {
   return (
     <div>
       <p className="help-talk">
-        For help filling out this form,
-        <br />
-        ask the Education Call Center:
-      </p>
-      <p className="help-phone-number">
-        <a className="help-phone-number-link" href="tel:1-888-442-4551">
-          888-442-4551 (888-GI-BILL-1)
+        Call us at{' '}
+        <a href="tel:800-827-1000" aria-label="8 0 0. 8 2 7. 1 0 0 0.">
+          800-827-1000
```

Comment:
Why the change from `888-442-4551` to `800-827-1000`? Please make sure this was an intended change.

Reply:
This was a change specified by the content team [in this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11550#issuecomment-664603173).

Reply:
Ok, as long as @DanielleThierryUSDSVA is aware.